src/tuning/kernels/xgemv.cpp

```diff
@@ -52,7 +52,14 @@ class TuneXgemv {
   static size_t DefaultBatchCount() { return 1; } // N/A for this kernel
   static double DefaultFraction() { return 1.0; } // N/A for this kernel
   static size_t DefaultNumRuns() { return 10; } // run every kernel this many times for averaging
-
+  static size_t DefaultNumSearchStragegy() { return 1; } // N/A for this kernel
+  static size_t DefaultSwarmSizePSO() { return 8; } // N/A for this kernel
+  static double DefaultInfluenceGlobalPSO(){ return 0.1; }// N/A for this kernel
+  static double DefaultInfluenceLocalPSO(){ return 0.3; }// N/A for this kernel
+  static double DefaultInfluenceRandomPSO(){ return 0.6; }// N/A for this kernel
+  static size_t DefaultHeuristic(){ return size_t{0};} // Full search
+  static double DefaultMaxTempAnn(){ return 1.0;}// N/A for this kernel
```

Comment:
This block of code is duplicated quite a few times. It is not so nice, mostly because these parameters are not relevant at all to this routine. But I think that's a problem of the design of the tuning infrastructure.... I think we should make a parent class to all these tuners and have some default settings in there. But I can do that outside of this PR.